hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ContainerReplicaPendingOps.java

```diff
@@ -163,6 +176,9 @@ public void removeExpiredEntries(long expiryMilliSeconds) {
           if (op.getScheduledEpochMillis() + expiryMilliSeconds
               < clock.millis()) {
             iterator.remove();
+            pendingOpCount.put(op.getOpType(),
+                pendingOpCount.get(op.getOpType()) - 1);
+            updateMetrics(op.getOpType());
```

Comment:
Maybe rename this to updateTimeoutMetrics.

Also, is it worth passing the full Op in so we get the replica index in the updateMetrics and can apply the same `if replicaIndex > 0` logic as it other places?